mungegithub/mungers/submit-queue.go

```diff
@@ -159,8 +160,14 @@ type submitQueueInterruptedObject struct {
 type submitQueueMetadata struct {
 	ProjectName string
 
-	ChartUrl    string
-	HistoryUrl  string
+	ChartUrl   string
+	HistoryUrl string
+	// chartUrl and historyUrl are option storage locations. They are distinct from ChartUrl and
+	// HistoryUrl since the the public variables are used asynchronously by a fileserver and updates
+	// to the options values should not cause a race condition.
```

Comment:
This isn't worth the additional complexity. The race is benign-- at worst, someone loading the page could receive a slightly older version of the charturl/historyurl strings.